arcgis-ios-sdk-samples/Scenes/Orbit camera around object/README.md

```diff
@@ -0,0 +1,46 @@
+# Orbit camera around object
+
+Fix the camera to point at and rotate around a target object.
+
+![Image of orbit camera around object](orbit-camera-around-object.png)
+
+## Use case
+
+The orbit geoelement camera controller provides control over the following camera behaviors:
+
+* automatically track the target
+* stay near the target by setting a minimum and maximum distance offset
+* restrict where you can rotate around the target
+* automatically rotate the camera when the target's heading and pitch changes
+* disable user interactions for rotating the camera
+* animate camera movement over a specified duration
+* control the vertical positioning of the target on the screen
+* set a target offset (e.g.to orbit around the tail of the plane) instead of defaulting to orbiting the center of the object
+
+## How to use the sample
+
+The sample loads with the camera orbiting an airplane model. The camera is preset with a restricted camera heading and pitch, and a limited minimum and maximum camera distance set from the plane. The position of the plane on the screen is also set just below center.
+
+Use the sliders to adjust the camera heading and the plane's pitch. When not in Cockpit view, the plane's pitch will change independently to that of the camera pitch. Toggle on the switch to allow zooming in and out with the mouse/keyboard; when the switch is off, the user won't be able to adjust with the camera distance.
```

Comment:
```suggestion
Use the sliders to adjust the camera heading and the plane's pitch. When not in Cockpit view, the plane's pitch will change independently to that of the camera pitch. Toggle on the switch to allow zooming in and out with the mouse/keyboard; when the switch is off, the user won't be able to adjust the camera distance.
```
Should it be without "with"?

Reply:
I agree, remove the "with". Additionally, adjust the phrasing to suit mobile gestures.

```suggestion
Use the sliders to adjust the camera heading and the plane's pitch. When not in Cockpit view, the plane's pitch will change independently to that of the camera pitch. Toggle on the switch to allow zooming in and out by pinching; when the switch is off, the user won't be able to adjust with the camera distance.
```